scripts/init-database.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+
+values = list()
+
+
+def retreiveValues():
+    stream = open("../.config/bassa/config.yml", "r")
+    docs = yaml.load_all(stream)
+    global values
+    for doc in docs:
+        for k, v in doc.items():
+            for m, n in v.items():
+                values.append(n)
+
+
+def createDatabase():
+    param = values[5]+'://'+values[0]+':'+values[4]+'@'+values[2]
+    engine = sqlalchemy.create_engine(param)
+    engine.execute("CREATE DATABASE IF NOT EXISTS " + values[3])
+
+
+def importSQL():
+    param = values[5]+'://'+values[0]+':'+values[4]+'@'+values[2]+'/'+values[3]
+    engine = sqlalchemy.create_engine(param)
+    fd = open("../Bassa.sql", 'r')
+    sqlFile = fd.read()
+    fd.close()
+    sqlCommands = sqlFile.split(';')
+    for command in sqlCommands:
+        try:
+            with engine.connect() as con:
+                con.execute(command+';')
+        except:
+            pass
+retreiveValues()
```

Comment:
Keep some space before this block. How about we put these inside a Python `__main__` check

```
if __name__ == "__main__":
   createDatabase()
   importSQL()
   print "Database (" + values[3] + ") is now successfuly setup"
```